collect_app/src/androidTest/java/org/odk/collect/android/feature/smoke/AllWidgetsFormTest.java

```diff
@@ -548,6 +548,9 @@ public void testLikertWidget() {
     }
 
     public void testSelectOneFromMapWidget() {
+        onView(withText("Select place")).perform(click());
```

Comment:
What's the point of having this here?

Reply:
I've started adding these checks to `AllWidgetsFormTest` as it catches problems I've run into working on widgets that open a dialog/`Activity` from a button. For example, here you could unit test all the components involved, but have a some incorrect wiring between the widget and the dialog (like a missing arg) that gets opened pretty easily without having any tests fail. This checks that clicking the button at least doesn't crash.

Reply:
Ok I see.